src/langserver.ts

```diff
@@ -246,6 +243,12 @@ export function deactivateVls(): void {
 export function killVlsProcess(): void {
 	if (vlsProcess && !vlsProcess.killed) {
 		log('Terminating existing VLS process.');
-		terminate(vlsProcess);
+        // Not sure, how the terminate function works
```

Comment:
some lines are indented with tabs, some are with spaces. Could you run a formatter like prettier on it?